test/rpc/cart_rpc_one_node.yaml

```diff
@@ -36,13 +36,14 @@ tests: !mux
     cli_ppn: "1"
   echo:
     name: echo_basic
+    logparse: "1"
     srv_bin: tests/crt_echo_srv
     srv_arg: ""
-    srv_env: ""
+    srv_env: "-x D_LOG_FILE_APPEND_PID=1"
```

Comment:
I think a better place to set this env var is in log_check when logparse=1. With the way it is now, you'll need to remember to set logparse=1 AND set the env var, else it won't work correctly

Reply:
okay